docker-compose.yml

```diff
@@ -8,6 +8,15 @@ services:
     volumes:
       - /var/lib/postgresql
     command: echo true
+    
+  nginx:
+    build: .
+    ports:
+        - "80:80"
```

Comment:
@shivamMg Alright! I'll ping this issue on [Gitter](https://gitter.im/fossasia/open-event-orga-server).